build.gradle

```diff
@@ -66,9 +66,9 @@ dependencies {
     implementation('net.jpountz.lz4:lz4:1.3.0') {
         because('cassandra driver requires lz4 impl in classpath for compression')
     }
-    implementation 'com.exactpro.th2:grpc-data-provider:1.1.0'
+    implementation 'com.exactpro.th2:grpc-data-provider:1.1.0-TS-1083-response-formats-2759013731-SNAPSHOT'
 
-    implementation 'io.github.microutils:kotlin-logging:2.1.14'
+    implementation 'io.github.microutils:kotlin-logging:2.1.23'
```

Comment:
Please, return the previous version back. The new one brings the version of kotlin 1.6.0 but we are using 1.5.32